docs/en/resources/tools/postgres/postgres-list-publication-tables.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "postgres-list-publication-tables"
+type: docs
+weight: 1
+description: >
+ The "postgres-list-publication-tables" tool lists publication tables in a Postgres database.
+aliases:
+- /resources/tools/postgres-list-publication-tables
+---
+
+## About
+
+The `postgres-list-publication-tables` tool lists all publication tables in the database. It's compatible with any of the following sources:
+
+- [alloydb-postgres](../../sources/alloydb-pg.md)
+- [cloud-sql-postgres](../../sources/cloud-sql-pg.md)
+- [postgres](../../sources/postgres.md)
+
+`postgres-list-publication-tables` lists detailed information as JSON for publication tables. The tool takes the following input parameters:
```

Comment:
Nit: can we add a description on what publication tables are? Or can we add a link to an official doc page?